ELLog/Destinations/Crashlytics/LogCrashlyticsDestination.swift

```diff
@@ -22,14 +22,9 @@ The default behavior is:
     showTimestamp = false
 */
 @objc(ELLogCrashlyticsDestination)
-public class LogCrashlyticsDestination: LogDestinationBase, LogDestinationProtocol {
+public class LogCrashlyticsDestination: LogDestinationBase {
 
-    // LogDestinationProtocol compliance
-    public var showCaller: Bool = true
-    public var showLogLevel: Bool = false
-    public var showTimestamp: Bool = false
-
```

Comment:
Removal of this code changes behavior. The defaults are different than in LogDestinationBase. Also, make sure this code compiles in walmart-core, because that code includes this source file, whereas ELLog does not (due to Crashlytics reliance)